### ISSUE TITLE
E121: Undefined variable: v:false (Vim < 7.4.1154)

### ISSUE THREAD

**Issue by wookayin** [python-mode/python-mode#807]
The constant `v:false` is introduced in vim 7.4.1154, and it is being used since 3e37c3c343. By consequences, an older version of vim than 7.4.1154 raises the following error:

```
Error detected while processing ~./vim/plugged/python-mode/plugin/pymode.vim:
line  174:
E121: Undefined variable: v:false
E116: Invalid arguments for function pymode#default
```

Using the constant is good, but it is too recent. I think we'd better have a generous compatibility for the somewhat older versions of vim. If you agree on this, I can prepare a straightforward PR that replaces all the usage `v:false` by `0`.

**Comment by fmv1992** [comment]
Thanks mate. Just pushed to `develop` branch.

: )

**Comment by wookayin** [comment]
Thanks! It is now fixed in 43c052e.
